flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerFactory.java

```diff
@@ -56,6 +59,33 @@ protected Configuration getEffectiveConfigurationForResourceManagerAndRuntimeSer
                 configuration, TaskManagerOptions.TOTAL_PROCESS_MEMORY);
     }
 
+    @Override
+    protected Configuration getEffectiveConfigurationForResourceManager(
```

Comment:
We can add a test for it to verify:
- If the fine-grained resource management is enabled, those options should be removed.
- If the fine-grained resource management is not enabled, those options should be passed by.